### PR TITLE
PAYARA-2912 JBatch not working with PostgreSQL

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java [hunk 1]
```diff
@@ -521,8 +521,8 @@ private boolean isSimpleMode() {
         String columnID = "jobinstanceid";
         try (Connection connection = dataSource.getConnection()) {
             try (PreparedStatement preparedStatement = connection.prepareStatement(query)) {
-                preparedStatement.setString(1, parameter1);
-                preparedStatement.setString(2, parameter2);
+                preparedStatement.setInt(1, Integer.parseInt(parameter1));
```
Comment (smillidge) [inline appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java]: You will need to test this change with other database servers

Comment (Cousjava) [reply appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java]: Tested with PostgreSQL, Derby and MySQL

Comment (MeroRai) [reply appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java]: I tested it with Derby

Comment (Cousjava) [reply appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java]: Also tested with H2

Comment (Pandrex247) [reply appserver/batch/glassfish-batch-commands/src/main/java/org/glassfish/batch/ListBatchJobs.java]: Oracle DB is the last big'un to test against then

**Pull request merged**